lib/StaticUniformGrid.js

```diff
@@ -0,0 +1,277 @@
+'use strict';
+var Cesium = require('cesium');
+var AxisAlignedBoundingBox = Cesium.AxisAlignedBoundingBox;
+var Cartesian3 = Cesium.Cartesian3;
+var clone = require('clone');
+
+module.exports = StaticUniformGrid;
+
+/**
+ * A Uniform Grid for unmoving objects.
+ * @alias StaticUniformGrid
+ * @constructor
+ *
+ * @param {Object[]} objects Objects that the grid should store.
+ * @param {Function} aabbFunction A function for checking the axis aligned bounding box of an object in space.
```

Comment:
For an example of documenting callbacks, see https://github.com/AnalyticalGraphicsInc/cesium/blob/03191f3c7558d8ab6b3fe6fb002261501a2d9eb6/Source/DataSources/CallbackProperty.js#L106-L113